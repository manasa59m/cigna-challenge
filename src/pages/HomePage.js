import React from 'react';
import ListComponent from '../components/list/ListComponent';

const HomePage = () => {
    return (
        <div>
            <ListComponent />
        </div>
    )    
}

export default HomePage;
