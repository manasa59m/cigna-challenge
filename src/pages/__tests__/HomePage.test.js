import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import HomePage from "../HomePage";

describe("HomePage Component", () => {
  it("should render without crash", () => {
    const wrapper = shallow(<HomePage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
