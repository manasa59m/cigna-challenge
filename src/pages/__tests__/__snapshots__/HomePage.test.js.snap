// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render without crash 1`] = `
<div>
  <ListComponent />
</div>
`;
